packages/bsky/src/api/app/bsky/notification/listNotifications.ts

```diff
@@ -15,11 +15,11 @@ import {
 import { Notification } from '../../../../proto/bsky_pb'
 import { uriToDid as didFromUri } from '../../../../util/uris'
 import { Views } from '../../../../views'
-import { clearlyBadCursor, resHeaders } from '../../../util'
+import { resHeaders } from '../../../util'
 
 export default function (server: Server, ctx: AppContext) {
   const listNotifications = createPipeline(
-    skeleton,
+    skeleton(ctx.cfg.notificationsDelayMs),
```

Comment:
We don't need to wrap `skeleton` in a function like this, as `ctx` is already plumbed through the pipeline.  You'll just want to reflect `cfg` on the `Context` type.  See `getPostThread` for an example.